web/src/i18n/locales/en.ts

```diff
@@ -64,7 +64,9 @@ const en = {
       frameDetect: 'Frame Detect',
       frameDetectTip:
         "Calculate the difference between frames. Stop transmitting video stream when no changes are detected on the remote host's screen.",
-      resetHdmi: 'Reset HDMI'
+      resetHdmi: 'Reset HDMI',
+      enableHdmi: 'Enable HDMI',
```

Comment:
Could it be better to have "enable" and "disable" as the locale's to avoid the duplication here?